lib/l10n/intl_es.arb

```
@@ -894,6 +899,60 @@
+  "retry": "Reintentar",
+  
+  "@_comment_dispute_info": "Texto de Información de Disputas",
+  "unknown": "Desconocido",
+  "disputeWith": "Disputa con {role}: {counterparty}",
+  "@disputeWith": {
+    "placeholders": {
+      "role": { "type": "String" },
+      "counterparty": { "type": "String" }
+    }
+  },
+  "orderIdLabel": "ID de Orden",
```

RuboCop (1.76.1)

[warning] Duplicated key in hash literal.

(Lint/DuplicateHashKey)